### DISCUSSION TITLE
Move sub-packs in shop to different collections

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#424] [Suggestions]
Hey Max,

I am currently doing an overhaul of my shop and I am finding I have to do a lot of copying and pasting to move sub-packs around. Would it be possible to have the ability to move sub-packages between shop collections? 

Thank you,
Gem

**Comment by mvaessen** [comment]
Hi Gem,

That is definitely something I can add. This week i'm expecting to be mostly busy with the backend system upgrades, but after that I will take a look!
